packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -180,6 +187,7 @@ const errors = {
     `),
     type: Type.GRAPHQL,
     level: Level.ERROR,
+    category: ErrorCategory.USER,
```

Comment:
This is debatable but we should have a lookout for this error as it's could be gatsby-theme/gatsby-plugin related too.